x-pack/plugins/security/server/authentication/providers/basic.ts

```diff
@@ -86,7 +87,7 @@ export class BasicAuthenticationProvider extends BaseAuthenticationProvider {
         state: authHeaders,
       });
     } catch (err) {
-      this.logger.debug(`Failed to perform a login: ${err.message}`);
+      this.logger.debug(`Failed to perform a login: ${getDetailedErrorMessage(err)}`);
```

Comment:
Right, I thought it would be okay because we log all `AuthenticationResult.failed()` with `ERROR` level in the authentication service (for both `authenticate` and `login`). It's arguably still handy to have these `DEBUG` logs here for easier debugging when we enable debug mode (e.g., log messages will include `basic` as part of the context - we can still derive this info from authentication service logs, but it's a bit harder).

Let me know if you prefer to have `ERROR` directly in the providers as well (I did this initially, but then reverted in favor of logs in the authentication service).

Reply:
Ok, I suspected that. Thanks for clarifying. I think leaving them as debug here is fine.